src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -184,3 +184,8 @@
 
 CONST_AZURE_KEYVAULT_NETWORK_ACCESS_PUBLIC = "Public"
 CONST_AZURE_KEYVAULT_NETWORK_ACCESS_PRIVATE = "Private"
+
+# refer https://api.github.com/repos/Azure/draft/releases/latest
+# tag_name gives latest version released.
+# Moving away from 1:n release to avoid unwanted breaking changes with auto upgrades.
+CONST_DRAFT_CLI_VERSION = "v0.0.23"
```

Comment:
Is there any new features that must use version `v0.0.23`? Could we set the version to `v0.0.22` to avoid breaking change?

Reply:
💡Thats correct, more like long term fix and moving away from `1:n` release dependency, which has caused current failing tests for this because it automatically downloaded latest. 

**Extra Details:**

Every release version should respect the contract of stable released underlying `draft cli`.  Currently this feature follow `1:n` release which is causing issue, we want to lock it to the draft version so that new `Draft cli` release versions magically don't break the existing `az cli` 

FYI cc: @qpetraroia, @gambtho, @karenychen, @sabbour.

Reply:
Got it, thanks~